src/argilla/client/api.py

```diff
@@ -235,6 +238,22 @@ async def log_async(
     return await asyncio.wrap_future(future)
 
 
+def _check_if_feedback_dataset_exists(client: Argilla, name: str, workspace: str) -> bool:
+    response = list_workspaces_api_v0(client.http_client.httpx)
+    workspace_id = None
+    for ws in response.parsed:
+        if ws.name == workspace:
+            workspace_id = ws.id
+            break
+
+    response = list_datasets_api_v1(client.http_client.httpx)
+    for dataset in response.parsed:
+        if dataset.name == name and dataset.workspace_id == workspace_id:
+            return True
+
+    return False
```

Comment:
There is a similar function `argilla.client.feedback.utils.feedback_dataset_in_argilla` but it was impossible to use it in this module because it generates cyclic imports errors. I tried to fix them, but that would have required moving a lot of stuff... so I decided that it's not worth it and create this function instead.